tools/mksurfdata_map/Makefile.data

```diff
@@ -128,6 +129,12 @@ global-present-ne16np4 : FORCE
 global-present-ne120np4 : FORCE
 	$(MKSURFDATA) -no-crop -glc_nec 10 -y 2000 -res ne120np4 $(BACKGROUND)
 
+# BUG(wjs, 2019-04-10, ESCOMP/ctsm#262) This -hirespft won't work until issue 262 is resolved
+# For now, I'm using:
+#    -no-crop -glc_nec 10 -y 2005 -res 0.125nldas2
+global-present-nldas : FORCE
+	$(MKSURFDATA) -no-crop -hirespft -glc_nec 10 -y 2005 -res 0.125nldas2 $(BACKGROUND)
```

Comment:
I debated whether to write this rule according to what I used for the current dataset or according to what we eventually want. I decided on the latter, with the assumption that #262 will be fixed relatively soon, because I was concerned that, if I wrote it the other way, we'd forget to change it when it should be changed. But I'm open to changing this if you'd like.

Reply:
I agree with this one.